docs/changelog/1_0_a7.rst

```diff
@@ -0,0 +1,86 @@
+===========
+1.0 Alpha 7
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 7 "Lalande".
+
+
+EdgeQL
+======
+
+* Implement consistent update of expressions when referenced schema
+  objects are renamed (:eql:gh:`#1841`).
```

Comment:
Overall this looks like you simply copy/pasted git log here. We need more editing, grouping of related commit messages, and some examples.